ansible/collectory-standalone.yml

```diff
@@ -1,8 +1,8 @@
 - hosts: collectory
   roles:
-    - common
-    - java    
-    - mysql
-    - tomcat    
-    - apache
-    - collectory
+    - { role: common, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: java, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: mysql, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: tomcat, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: apache, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: collectory, collectory: ala-collectory, collectory_version: 1.0 }
```

Comment:
Interesting. Looks like it's just the version that is used by common. This would explain why the biocache stuff uses "LATEST" for the version.

Rather than parameterising every role in the playbook, could you add a default value to the version variable in roles/collectory/vars/main.yml?
```
version: "{{ collectory_version | default('LATEST') }}"
```